core/openapi.yaml

```diff
@@ -605,6 +553,8 @@ components:
           minItems: 1
           items:
             $ref: "#/components/schemas/TrainSchedule"
+        trains_path:
+        ######## ask about it
```

Comment:
Here I am :)

Reply:
I did not find the corresponding class in the code to get the details of trains_path.
There is also a similar comment related to 'modes' in rollingStock